cypress/e2e/awx/resources/executionEnvironmentsAccess.cy.ts

```diff
@@ -90,8 +91,9 @@ describe('Execution Environments: User/Team access', () => {
     cy.clickButton(/^Close$/);
   });
 
-  // TODO: Skipped until bug AAP-27071 is addressed
+  // There's inconsistency with Users -> creating a team with an org doesn't show it under the org so skipping for time being
```

Comment:
Have we filed an issue for this?

Reply:
Here's the issue https://issues.redhat.com/browse/AAP-27980